### DISCUSSION TITLE
m2000c bug

### DISCUSSION THREAD

**Discussion by bomberns127** [akaAgar/briefing-room-for-dcs#588] [General]
hello, i have created a few missions for the m2000c, i then open dcs, go to missions, choose the saved mission, and after the initial load i then press fly i get 'mission cannot be saved due to errors'
along with 'enfield 3' then' invalid frequency 124 mhz'
i then created a mission for the su27 and it went through no issues

**Comment by c3f59** [comment]
I've had something similar, but not with m2000C. You can add your info to the Issue https://github.com/akaAgar/briefing-room-for-dcs/issues/637
